test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1630,7 +1651,8 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
         .then(function (test) {
           test.form.facebookButton().click();
           return tick(test);
-        }).then(function(test) {
+        })
+        .then(function(test) {
```

Comment:
I could not get this to work with **waitForSpyCall**. It caused wait condition failures. I had to leave it as **tick()**.